source/mesh_deformation/interface.cc

```diff
@@ -624,9 +624,15 @@ namespace aspect
       this->get_signals().post_compute_no_normal_flux_constraints(sim.triangulation);
 
       // Ask all plugins to add their constraints.
-      // For the moment add constraints from all plugins into one matrix, then
+      // For the moment add constraints from all plugins into one object, then
       // merge that matrix with the existing constraints (respecting the existing
-      // constraints as more important)
+      // constraints as more important).
+      //
+      // We initialize the constraints object with the locally
```

Comment:
I think this is incorrect. According to https://www.dealii.org/developer/doxygen/deal.II/classAffineConstraints.html#a761e5aa073046b19879ba8ff7dd3ae56 one should use locally_owned for the first argument.

Reply:
Let me do this as a follow-up, so that this patch really only documents the current state.

Reply:
See #6632.